src/adaptors/graphql/index.js

```diff
@@ -0,0 +1,82 @@
+import { createStore, combineReducers } from 'redux';
+import { graphql } from 'graphql';
+
+import parseSchema from './utils/parseSchema';
+import normalize from './utils/normalize';
+import merge from './utils/merge';
+
+const UPDATE_CACHE = 'UPDATE_CACHE';
+
+export default function createAdaptor(endpoint, schema) {
+  const parsedSchema = parseSchema(schema);
+  const reducers = Object.keys(parsedSchema).reduce((acc, key) => {
+    return {
+      ...acc,
+      [key]: createReducer(key),
+    };
+  });
+  const reducer = combineReducers(reducers);
+  const store = createStore(reducer);
```

Comment:
I really think that Adrenaline shouldn't be constructing stores. This approach locks your application out of utilizing redux for other purposes. The Adrenaline components can expect 'store' as context from a react-redux [Provider](https://github.com/rackt/react-redux/blob/master/docs/api.md#provider-store) and pass a reference into Adaptor methods. Similarly, the Adaptor can expose a reducer function so that applications can combineReducers and compose Adrenaline as needed.